### PR TITLE
Benitlux report tool

### PR DESCRIPTION
**Pull request by xymus**
Intro a tool to print a pretty report on the availability of beers at the excellent Brasserie Bénélux.

### CODE REVIEW

File: contrib/benitlux/src/report.nit [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+end
+
+# Sort beers by their availability and first date of appearance
+var availability = new HashMap[Beer, Int]
+var appearances = new HashMap[Beer, String]
+for beer in beers do
+	var days = beer2days[beer]
+	if days.not_empty then
+		appearances[beer] = days.first
+		availability[beer] = 1000-days.length
```
Comment (privat) [inline contrib/benitlux/src/report.nit]: 1000?

File: contrib/benitlux/src/report.nit [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+var days_sample = [for i in column_width.times do all_days[i*all_days.length/column_width]]
+
+for beer in beers do
+	var days = beer2days[beer]
+
+	# Skip never-available beers, usually name errors
+	if days.is_empty then continue
+
+	# Print a line looking like: "  ############ ######         ######## : Bieer"
+	printn ""
```
Comment (privat) [inline contrib/benitlux/src/report.nit]: useless

File: contrib/benitlux/src/report.nit [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+	# 1st sorting priority
+	var map1: HashMap[COMPARED, Comparable]
+
+	# 2nd sorting priority
+	var map2: HashMap[COMPARED, Comparable]
+
+	# Key compare
+	redef fun compare(a, b) do return if map1[a] == map1[b] then
+	                                      map2[a] <=> map2[b]
+	                                  else map1[a] <=> map1[b]
```
Comment (privat) [inline contrib/benitlux/src/report.nit]: ಠ_ಠ

Comment (Morriar) [inline contrib/benitlux/src/report.nit]: Buzzfeed would be like "Pretty printers hate him! Click here to know why.".

Comment (R4PaSs) [inline contrib/benitlux/src/report.nit]: ಠ_ಥ

**Pull request merged**